src/Maestro/Maestro.Web/Api/v2019_01_16/Models/Build.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using JetBrains.Annotations;
+
+namespace Maestro.Web.Api.v2019_01_16.Models
+{
+    public class Build : Maestro.Web.Api.v2018_07_16.Models.Build
```

Comment:
I think this model shouldn't inherit the previous. You need to remove some of the fields.

Reply:
Great. Will change it.